### DISCUSSION TITLE
Ability to collate photos by month

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#881] [Enhancements]
Currently photos are clubbed by the day.

_(Screenshots courtesy https://github.com/ente-io/photos-app/issues/1599)_
![image](https://github.com/ente-io/photos-app/assets/1161789/4c28c825-fafd-4ded-8a21-5e8c930b7f61)

In comparison, Google Photos shows 50% more images on the same page while enlarging the preview size dynamically:

![image](https://github.com/ente-io/photos-app/assets/1161789/66f856a0-ed6b-4e74-ab7c-dbef3f458703)

Having an option to collate photos by month, or dynamically or with a pinch-to-zoom interaction will make better use of the available screen real estate.

**Comment by harrrrshit** [comment]
I like the day wise view more, but it wouldn't be bad to have an option to change it. Just please don't make it compulsory and only view.

**Comment by KestrelWild** [comment]
I would love to see this too!
